build_system/seqan3-config.cmake

```
@@ -136,45 +136,36 @@ macro (seqan3_config_error text)
 endmacro ()
 
 # ----------------------------------------------------------------------------
-# Detect if we are a clone of repository and if yes auto-add submodules
+# Find SeqAn3 include path
 # ----------------------------------------------------------------------------
+
 # Note that seqan3-config.cmake can be standalone and thus SEQAN3_CLONE_DIR might be empty.
 find_path (SEQAN3_CLONE_DIR NAMES build_system/seqan3-config.cmake HINTS "${CMAKE_CURRENT_LIST_DIR}/..")
+find_path (SEQAN3_INCLUDE_DIR NAMES "seqan3/version.hpp" HINTS "${SEQAN3_CLONE_DIR}/include")
+find_path (SEQAN3_SUBMODULES_DIR NAMES submodules/range-v3 HINTS "${SEQAN3_CLONE_DIR}" "${SEQAN3_INCLUDE_DIR}/seqan3")
```

Shouldn't we use SDSL instead of Ranges for finding the the submodules, in case we drop the ranges dependency in the future? 

```suggestion
find_path (SEQAN3_SUBMODULES_DIR NAMES submodules/sdsl-lite HINTS "${SEQAN3_CLONE_DIR}" "${SEQAN3_INCLUDE_DIR}/seqan3")
```

---

Yes, true :)